### ISSUE TITLE
Sample ratio

### ISSUE THREAD

**Issue by chenshi3** [microsoft/SoftTeacher#112]
Hi,
I found sample_ratio=[1, 4] in your config file. Dose it mean that the weight of labeled data is 4?

**Comment by MendelXu** [comment]
No. It is just to sample different numbers of samples for labeled data and unlabeled data.

**Comment by chenshi3** [comment]
I mean in the dataloader the sample weight of labeled data is 1 and the sample weight of unlabeled data is 4.

**Comment by MendelXu** [comment]
Yes, it is. However, it is not implemented in a probabilistic sampling way.
